src/core_ocean/shared/mpas_ocn_tracer_hmix_redi.F

```diff
@@ -143,7 +143,7 @@ subroutine ocn_tracer_hmix_redi_tend(meshPool, scratchPool, layerThicknessEdge,
          dTracerdZTopOfCell, dTracerdZTopOfEdge, areaCellSum
 
       type (field2DReal), pointer :: gradTracerEdgeField, gradTracerTopOfEdgeField, gradHTracerSlopedTopOfCellField, &
-                                     dTracerdZTopOfCellField, dTracerdZTopOfEdgeField, areaCellSumField
```

Comment:
@mark-petersen, this change needs to be reverted.  Maybe this is what your comment above refers to, but `areaCellSumField` is needed.  The current code never allocates `areaCellSum` but assigns to it.  Bad news!